pkg/kapp/diff/change_set_with_versioned_rs.go

```diff
@@ -13,7 +13,8 @@ import (
 )
 
 const (
-	versionedResAnnKey        = "kapp.k14s.io/versioned" // Value is ignored
+	versionedResAnnKey        = "kapp.k14s.io/versioned"               // Value is ignored
+	versionedResOrigAnnKey    = "kapp.k14s.io/versioned-keep-original" // Value is ignored
```

Comment:
thinking on a better name for "versioned-keep-original"...

Reply:
Since we are trying to keep the previous version, we can call it `version-keep-previous`?
```suggestion
	versionedResOrigAnnKey    = "kapp.k14s.io/version-keep-previous" // Value is ignored
```

Reply:
Or, since we are not updating the version, we can call it `version-no-update`?
```suggestion
	versionedResOrigAnnKey    = "kapp.k14s.io/version-no-update" // Value is ignored
```